merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -163,6 +163,18 @@ export class Windows {
     });
   }
 
+  /**
+   * Selects the ith true window
```

Comment:
Please add a bit that says it falsifies all the other true segments. Otherwise its unclear, could be interpreted as deleting the other segments.

Reply:
done